recipes/libcxxwrap-julia/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "libcxxwrap-julia" %}
+{% set version = "0.2.2" %}
+{% set sha256 = "62cb4216efb3dc39736630ec747fb35e8685e6cd888b65d6e431905ba45918c6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/JuliaInterop/libcxxwrap-julia/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # Julia has not been packaged for conda on OS X or Windows
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+    - julia >=0.6,<0.7
+  run:
+    - julia >=0.6,<0.7
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/jlcxx  # [unix]
+    - test -f ${PREFIX}/include/jlcxx/jlcxx.hpp  # [unix]
+    - test -f ${PREFIX}/lib/cmake/JlCxx/JlCxxConfig.cmake  # [unix]
+    - test -f ${PREFIX}/lib/cmake/JlCxx/JlCxxConfigVersion.cmake  # [unix]
+
+    - if exist %LIBRARY_PREFIX%\include\JlCxx\jlcxx.hpp (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\JlCxx\JlCxxConfig.cmake (exit 0) else (exit 1)  # [win]
+    - if exist %LIBRARY_PREFIX%\lib\cmake\JlCxx\JlCxxConfigVersion.cmake (exit 0) else (exit 1)  # [win]
```

Comment:
These don't really matter if it is a linux-only package, but no harm in keeping them around